_posts/2020-07-05-emoji.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Emoji in alt text
+cases:
+  - title: Face example
+    example: "Runner at finish line, looking 😤"
+    macos: emoji-01-macos-voiceover-10-15.mp3
+    jaws: emoji-01-windows-jaws-2020.mp3
+    nvda: emoji-01-windows-nvda-2019-2-1.mp3
+---
+
+This test details audible effects of emoji (in alternative text) when read by different screen readers. Inspired by [Emoji Accessibility for Visually Impaired People](https://dl.acm.org/doi/10.1145/3313831.3376267). See [the specific test case from the research article on YouTube](https://youtu.be/uIbPcZq6izk?t=480).
+
+{% include table.html cases=page.cases %}
+
+## Findings
+
+VoiceOver for macOS describes the emoji as angry — "looking, huffing in anger face". JAWS describes the emoji as triumphant — "looking face with look of triumph". NVDA describes the emoji more objectively — "looking face with steam from nose".
```

Comment:
@jsnmrs What do you think about adding:

> Emoji descriptions can vary between operating systems and software. 

The finding allude to it, but I think it could be helpful to spell it out.

Possibly even add:

> The spoken description of the emoji may not match your visual intention.

Reply:
Thanks, that's a good idea. I sandwiched that language around the existing findings.